lerna.json

```diff
@@ -1,15 +1,11 @@
 {
   "lerna": "3.4.3",
-  "packages": [
-    "packages/*"
-  ],
   "version": "8.1.3",
+  "npmClient": "pnpm",
+  "useWorkspaces": true,
   "command": {
-    "bootstrap": {
-      "npmClientArgs": [
-        "--no-package-lock",
-        "--include=dev"
-      ]
+    "publish": {
```

Comment:
Can we get rid of Lerna completely? I wonder how much value we get from it at this point.

Reply:
pnpm mentions a couple options for handling the release workflow (versioning, publishing, etc): https://pnpm.io/workspaces#release-workflow

Reply:
Hi Webdriver Folks 👋 

I'm James, you might possibly know me already from having created https://github.com/typescript-eslint/typescript-eslint or my work on some other popular open-source tools over the years such as ESLint, Prettier and Babel. Right now I've got my Lerna maintainer hat on 🐉  

There's a few things I'd love to dig into here in detail:

✅  Removing bootstrap

I'm delighted to see this PR to modernize your Lerna setup. `lerna bootstrap` was a uniquely useful tool back when it was created because no package managers (only npm existed at the time) supported package managers natively. We are deprecating it in v7 around May time.

🙅 Removing lerna entirely

The key point I want to emphasise - lerna has full support for pnpm, so there is no reason that leveraging pnpm should prompt a move away from lerna.

`lerna version` and `lerna publish` are still hugely valuable and set the standard that other tools like Changesets seek to emulate. We took over stewardship of lerna only last summer but we have already landed many features and QOL improvements, and we have many more planned. The project is alive and well and now that we have raised funding we have much more time to spend on making it a success.

An extra, critical, consideration is that lerna's task runner can be used to augment these npm/pnpm scripts with caching - for free. If you run any deterministic script such as build, test, lint etc more than once, then you restore that full task output instantly from the cache, including any artifacts that may have been created by the task. This can lead to truly massive saving in CI runs, in particular.

Additionally, the lerna task runner understands how all of your packages and tasks relate to one another, so you can specify advanced task pipelines - things like having test depend on build, build depend on the build task of dependency packages - the possibilities are huge.

I would be delighted to spend a day getting to know your repo and setting you up with an optimal lerna setup including all of the above goodies around task caching. Perhaps once this PR gets merged, if that's the general plan?

Another totally optional thing - if you have a paid team Slack account, we would also be delighted to leverage Slack Connect between our workspaces to allow for frictionless ongoing communication. If that's something you are interested in, please send me a DM with your relevant Slack details: https://twitter.com/MrJamesHenry

If you have a similar chat tool you would like me to join instead, then I am definitely open to that.

Cheers!

James

Reply:
^ @christian-bromann

Reply:
> lerna version and lerna publish are still hugely valuable and set the standard that other tools like Changesets seek to emulate.

I totally agree with this statement. The current release process with autogenerated release notes is very smooth.

Reply:
Awesome, then I’ll keep watching this PR for when it goes in and spend the time reviewing and optimizing the lerna side once it’s merged. I’ll then provide a video walkthrough of the changes and suggestions so that you can review with full context.

Again, any questions or feedback in the meantime please don’t hesitate to reach out!

Reply:
I haven't been able to find a solution to the Windows CI failures. Once that is resolved this PR should be ready for review.